src/admin-views/aggregator/status.php

```diff
@@ -198,14 +199,14 @@
 			$notes .= '</a>';
 		}
 		?>
-        <tr>
-            <td class="label">
-                <img src="<?php echo esc_url( tribe_events_resource_url( 'images/aggregator/meetup.png' ) ); ?>" /><span><?php esc_html_e( 'Meetup', 'the-events-calendar' ); ?></span>
-            </td>
-            <td class="indicator <?php esc_attr_e( $indicator ); ?>"><span class="dashicons dashicons-<?php echo esc_attr( $indicator_icons[ $indicator ] ); ?>"></span></td>
-            <td><?php echo esc_html( $text ); ?></td>
-            <td><?php echo $notes; // Escaping handled above ?></td>
-        </tr>
+		<tr>
+			<td class="label">
+				<img src="<?php echo esc_url( tribe_events_resource_url( 'images/aggregator/meetup.png' ) ); ?>" /><span><?php esc_html_e( 'Meetup', 'the-events-calendar' ); ?></span>
+			</td>
+			<td class="indicator <?php esc_attr_e( $indicator ); ?>"><span class="dashicons dashicons-<?php echo esc_attr( $indicator_icons[ $indicator ] ); ?>"></span></td>
+			<td><?php echo esc_html( $text ); ?></td>
+			<td><?php echo $notes; // Escaping handled above ?></td>
```

Comment:
:warning: Expected next thing to be an escaping function (see Codex for 'Data Validation'), not '$notes'

<sub>hash: 7a379186b842d72fdc4d7074ef508116</sub>